urllib3/poolmanager.py

```diff
@@ -39,10 +39,38 @@
     'HTTPSPoolKey', HTTPPoolKey._fields + SSL_KEYWORDS
 )
```

Comment:
Nope. I was going to leave them because I thought removing them would be API-breaking since they've been in a release, but if you're comfortable with them gone I'm happy to yank them out.